src/utils/dom.ts

```diff
@@ -1,6 +1,17 @@
 import { CSS_UTILITY } from "./resources";
 import { guid } from "./guid";
 
+export const findFormFieldLabel = (componentEl: HTMLElement): string | null => {
```

Comment:
Let's incorporate more of Ionic's [`getAriaLabel`](https://github.com/ionic-team/ionic-framework/blob/master/core/src/utils/helpers.ts#L140) util to consider aria attributes since that is a common pattern.